### DISCUSSION TITLE
how to add new database?

### DISCUSSION THREAD

**Discussion by zprofreelancer** [serenity-is/Serenity#5381] [Help]
Hello,
i need to add my own database beside the northwind database...how to add the database and how to add its link and my tables links in the navigation side bar?

Thanks.

**Comment by erehtolleh** [comment]
You need to change the Default connection strings or create a new one to connect to your local database.  Install sql server express.  Then follow the tutorial.
[movie tutorial](https://serenity.is/docs/tutorials/movies/creating_movie_table)

**Reply by zprofreelancer** [reply]
This will create only one table...can i  create multiple tables in the DefaultDB_20160519_115100_MovieTable.cs. file?

**Reply by erehtolleh** [reply]
yes create as many as you want

**Reply by zprofreelancer** [reply]
can i use the model view controller structure?....can i create controller folder and create controllers and the view pages that correspond to actions inside the controller file?

**Reply by erehtolleh** [reply]
Yes same as MVC, look at the Dashboard sample

**Reply by zprofreelancer** [reply]
yes..but in dashboard sample the controller and model and view are all in the same folder....can i use the default mvc structure?

**Reply by erehtolleh** [reply]
No.  I don't like that structure.  I like that they are all in the same place.  It is not configured to pick up by folder automatically...at least I have not tried to do it that way.

**Reply by zprofreelancer** [reply]
why the northwind tables pages doesn't follow mvc concept like the dashboard do?

**Reply by zprofreelancer** [reply]
for example the customer page, where is the cshtml customer with the new customer button?

**Reply by stixoffire** [reply]
Its in the dialog template.
Look in the project under shared  things - like templates - you can change it on the fly in code too .. The Serenity Model makes things easy and simple . Not difficult. 
Create your own data base , point sergen at it and let it generate millions of lines of code with the same look and feel all at once .. modify as you like ..
